### PR TITLE
descendant metrics

### PR DESCRIPTION
**Pull request by ethn**
not yet submitted for approval.

(majorly broken, no new specs...)

### CODE REVIEW

File: mod/homepage/template/self/homepage_top_banner.haml [hunk 1]
```diff
@@ -6,8 +6,7 @@
         = nest :wikirate_white_logo, view: :core, size: :medium
       .col-md-10
         %span.wikirate-logo-title
-          Wiki
-          %strong Rate
+          Wiki<strong>Rate</strong>
```
Comment (ethn) [inline mod/homepage/template/self/homepage_top_banner.haml]: fixes unwanted space

File: mod/metrics/set/type/metric.rb [hunk 2]
```diff
@@ -34,422 +37,31 @@ def metric_type_id
...
-  Answer.where(metric_id: id, company_id: company, year: year.to_i).take
-end
-
-def distinct_values
-  all_answers.select(:value).distinct.pluck(:value)
-end
-
-def question_card
-  field "Question", new: {}
-end
```
Comment (ethn) [inline mod/metrics/set/type/metric.rb:53]: moved to accessor

File: mod/metrics/set/type/metric.rb [hunk 3]
```diff
@@ -34,422 +37,31 @@ def metric_type_id
...
-        <div class="yinyang-row">
-          <div class="metric-item value-item ">
-            #{yield}
-            <div class="details"></div>
-          </div>
-        </div>
-        HTML
-      end
-    end
-  end
```
Comment (ethn) [inline mod/metrics/set/type/metric.rb:269]: deleted this

File: mod/metrics/set/type/metric.rb [hunk 4]
```diff
@@ -34,422 +37,31 @@ def metric_type_id
...
-    end
-    res
-  end
-
-  view :outliers do
-    outs = Savanna::Outliers.get_outliers prepare_for_outlier_search, :all
-    outs.inspect
-  end
-
-  def css
```
Comment (ethn) [inline mod/metrics/set/type/metric.rb:195]: deleted this

File: mod/metrics/set/type/metric.rb (hunk 2 above)
Comment (xithan) [reply mod/metrics/set/type/metric.rb:53]: Doesn't this change the name?

File: mod/homepage/template/self/homepage_top_banner.haml (hunk 1 above)
Comment (xithan) [reply mod/homepage/template/self/homepage_top_banner.haml]:
```haml
Wiki
%strong> Rate
```
[whitespace removal](http://haml.info/docs/yardoc/file.REFERENCE.html#whitespace-removal--and-)

File: mod/metrics/set/abstract/hybrid.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+def hybrid?
+  hybrid_card.checked?
+end
+
+format :html do
+  view :content_formgroup do
+    super() + field_nest(:hybrid, title: "Hybrid")
+  end
+end
```
Comment (xithan) [inline mod/metrics/set/abstract/hybrid.rb:9]: ❗️#772 changes this at its old place ❗️

File: mod/metrics/lib/formula.rb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# module for organizing Calculator and Input classes
+module Formula
+  def self.calculator_class content
+    [Translation, Ruby].find { |klass| klass.valid_formula? content } || Wolfram
+  end
+end
```
Comment (xithan) [inline mod/metrics/lib/formula.rb]: move to `metric_type/formula` ?

**Review by xithan — APPROVED** [review]

**Pull request merged**